frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -230,7 +230,10 @@ function FileManagerMenu:setUpdateItemTable()
     self.menu_items.cloud_storage = {
         text = _("Cloud storage"),
         callback = function()
-            local cloud_storage = CloudStorage:new{}
+            local cloud_storage = CloudStorage:new{
+                width = Screen:getWidth(),
```

Comment:
@Frenzie 
I think that CloudStorage remembers width and height at first run. When we change orientation (resolution) then CS uses old value.
@ghiastas 
Good catch. Thanks.